csrc/ops/arith.h

```diff
@@ -736,21 +742,36 @@ NVF_API TensorView* argsort(
 //! Grouped matrix multiplication
 //!
 //! Performs matrix multiplication on grouped sets of matrices using offsets
-//! to define variable-sized groups.
+//! to define variable-sized groups. This op computes:
+//!
+//! alpha * grouped_mm((mat1 * scale1), (mat2 * scale2), offsets) + bias * beta
 //!
 //! \param mat1 First set of matrices
 //! \param mat2 Second set of matrices
 //! \param offsets Offsets tensor defining group boundaries
 //! \param scale1 Scale tensor for mat1
 //! \param scale2 Scale tensor for mat2
+//! \param alpha Global Scaling factor for mat1@mat2
+//! \param bias Bias tensor
+//! \param beta Scale tensor for bias
+//! \param dtype Output dtype
```

Comment:
What does it mean when `dtype` (or `block_scaling_factor_dtype`) is null?

Reply:
added a comment.

dtype == null, we'll just output the same dtype as input matrix

block_scaling_factor_dtype cannot be empty when `out_block_scale_size != 0`